From 6ac85e8e78e60cb723d84beba4391bb05de91e18 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Tue, 12 Mar 2013 07:14:48 -0700
Subject: [PATCH 1294/1357] MR1: Remove stripping of scheme, authority from submit dir in support of viewfs.

  Reason: bug
  Author: Alejandro Abdelnur
  Ref: CDH-10922

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-0.20-security-patches@1077795 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 6dc476279ad119495993caad3b0648f1b20287f0)
---
 .../mapred/org/apache/hadoop/mapred/JobClient.java |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobClient.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobClient.java
index 2e2b21e..06fd369 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobClient.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobClient.java
@@ -779,7 +779,6 @@ public class JobClient extends Configured implements MRConstants, Tool  {
           " that directory");
     }
     submitJobDir = fs.makeQualified(submitJobDir);
-    submitJobDir = new Path(submitJobDir.toUri().getPath());
     FsPermission mapredSysPerms = new FsPermission(JobSubmissionFiles.JOB_DIR_PERMISSION);
     FileSystem.mkdirs(fs, submitJobDir, mapredSysPerms);
     Path filesDir = JobSubmissionFiles.getJobDistCacheFiles(submitJobDir);
-- 
1.7.0.4

